mu/app.py

```diff
@@ -191,7 +194,8 @@ def excepthook(*exc_args):
                 "e": error,  # error message
             }
             args = urllib.parse.urlencode(params)
-            webbrowser.open("https://codewith.mu/crash/?" + args)
+            if "MU_SUPPRESS_CRASH_REPORT_FORM" not in os.environ:
+                webbrowser.open("https://codewith.mu/crash/?" + args)
```

Comment:
This new environment var is used during automated testing to suppress the opening of a browser during exception handler testing.